test/integration/features/index.js

```diff
@@ -539,4 +545,718 @@ describe('retry policy', () => {
+      const to = 0;
+      fromHeight = 100000;
+      const utxo = await dapiClient.getUTXO(faucetAddress, from, to, fromHeight);
+
+      expect(spy.callCount).to.be.equal(1);
+      expect(utxo.items).to.have.lengthOf(0);
+      expect(utxo).to.be.deep.equal({
+        "from": from,
+        "fromHeight": fromHeight,
+        "items": [],
+        "to": 1,
+        "totalItems": 1
+      });
+    });
+
+    it('should getUTXO by address with params: 1 0 100', async function it() {
+      const from = 1;
+      const to = 0;
+      fromHeight = 100;
+      const utxo = await dapiClient.getUTXO(faucetAddress, from, to, fromHeight);
+
+      expect(spy.callCount).to.be.equal(1);
+      expect(utxo.items).to.have.lengthOf(0);
+      expect(utxo).to.be.deep.equal({
+        "from": from,
+        "fromHeight": fromHeight,
+        "items": [],
+        "to": 1,
+        "totalItems": 1
+      });
+    });
+
+    it('should getUTXO by address with params: 1 0 100 99', async function it() {
+      const from = 1;
+      const to = 0;
+      fromHeight = 100;
+      toHeight = 99;
+      const utxo = await dapiClient.getUTXO(faucetAddress, from, to, fromHeight, toHeight);
+
+      expect(spy.callCount).to.be.equal(1);
+      expect(utxo.items).to.have.lengthOf(0);
+      expect(utxo).to.be.deep.equal({
+        "from": from,
+        "fromHeight": fromHeight,
+        "items": [],
+        "to": 1,
+        "totalItems": 1
+      });
+    });
+
+    it('should getUTXO by address with params: 0 0 100 2000', async function it() {
+      const from = 0;
+      const to = 0;
+      fromHeight = 100;
+      toHeight = 2000;
+      const utxo = await dapiClient.getUTXO(faucetAddress, from, to, fromHeight, toHeight);
+
+      expect(spy.callCount).to.be.equal(1);
+      expect(utxo.items).to.have.lengthOf(1);
+      expect(utxo).to.be.deep.equal(
+        {
+          'totalItems': 1,
+          'from': 0,
+          "fromHeight": fromHeight,
+          'to': 1,
+          'items': [
+            {
+              'address': faucetAddress,
+              'txid': utxo.items[0].txid,
+              'outputIndex': 0,
+              'script': utxo.items[0].script,
+              'satoshis': 10000000000,
+              'height': utxo.items[0].height
+            }
+          ]
+        });
+    });
+
+    it('should getUTXO by address 1 0 100 2000', async function it() {
+      const from = 1;
+      const to = 0;
+      fromHeight = 100;
+      toHeight = 2000;
+      const utxo = await dapiClient.getUTXO(faucetAddress, from, to, fromHeight, toHeight);
+
+      expect(spy.callCount).to.be.equal(1);
+      expect(utxo.items).to.have.lengthOf(0);
+      expect(utxo).to.be.deep.equal({
+        "from": from,
+        "fromHeight": fromHeight,
+        "items": [],
+        "to": 1,
+        "totalItems": 1
+      });
+    });
+
+
+      it('should throw error when call getTransactionsByAddress without params', async function it() {
+        let err = '';
```

Comment:
Formatting seems to be off in some tests.

Reply:
in all our tests eslint formatting is off https://github.com/dashevo/dapi-client/blob/master/.eslintignore#L1